tests/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -292,8 +313,8 @@ commands:
         num_nodes=$(oc get nodes -l node-role.kubernetes.io/worker -o name | wc -l)
         # check the number of ovn-controller pods that are running
         num_pods=$(oc get pods -n $NAMESPACE -l service=ovn-controller -o name --field-selector=status.phase=Running | wc -l)
-        if [ "$num_nodes" != "$num_pods" ]; then
-          echo "Cluster has $num_nodes nodes but OVNController spawned $num_pods pods"
+        if [ "$num_nodes" -ne "$((num_pods/2))" ]; then
```

Comment:
so what if the number of pods is 5 when there are 2 nodes? your check will pass because / means integer division; but that would be wrong, no? I think you should instead multiply num_nodes by 2 and check against that.